### ISSUE TITLE
request: config to customize commit message

### ISSUE THREAD

**Issue by r-priyam** [favware/cliff-jumper#2]
**Is your feature request related to a problem? Please describe.**

Currently there is no way to customize the commit message. 

**Describe the solution you'd like**

Having `package-name` in the commit scope is not really needed when there is only one package.

Example -

```diff
- chore(package-name): release package-name@0.0.0
+ chore(release): 0.0.0
```

The above seems more descriptive when there is only one package in the repo. Since there's no such tool around in node environment (afaik), I guess this can be added to enhance the experience.

A config option can be added like - 
```json
{
    monorepo: false
}
```
If `monorepo` is set to false it can directly just use simple commit message to commit.

**Comment by favna** [comment]
I'll get around adding this probably in the weekend. Busy week this week.

**Comment by r-priyam** [comment]
> I'll get around adding this probably in the weekend. Busy week this week.

I can give a shot to implement it but just I am not sure on what option should be added there in config for this to happen, if you can point me to the correct name then ✌️

**Comment by favna** [comment]
Released in v1.2.0
